sfdo_template_helpers/fields/markdown.py

```diff
@@ -70,19 +66,36 @@ class MarkdownFieldMixin:
         "a",
     ]
 
-    markdown_attrs = {"img": ["src", "alt", "title"], "a": ["href", "alt", "title"]}
+    allowed_attrs = {"img": ["src", "alt", "title"], "a": ["href", "alt", "title"]}
+
+    property_suffix = "_html"
 
     description = _("Field containing Markdown formatted text.")
 
     def __init__(self, *args, **kwargs):
-        self.property_suffix = kwargs.pop("property_suffix", DEFAULT_SUFFIX)
+        # an additional property with this suffix is added to the model
+        # to render the field, the default value is _html
+        self.property_suffix = kwargs.pop(
+            "property_suffix", MarkdownField.property_suffix
+        )
+        # declare whether or not the html property should be marked safe
+        # for the django template system
+        self.is_safe = kwargs.pop("is_safe", True)
+        self.allowed_tags = kwargs.pop("allowed_tags", MarkdownField.allowed_tags)
+        self.allowed_attrs = kwargs.pop("allowed_attrs", MarkdownField.allowed_attrs)
         super().__init__(*args, **kwargs)
 
     def deconstruct(self):
         name, path, args, kwargs = super().deconstruct()
         # Only include kwarg if it's not the default
-        if self.property_suffix != DEFAULT_SUFFIX:
+        if self.property_suffix != MarkdownField.property_suffix:
             kwargs["property_suffix"] = self.property_suffix
+        if not self.is_safe:
+            kwargs["is_safe"] = self.is_safe
+        if self.allowed_tags != MarkdownField.allowed_tags:
+            kwargs["allowed_tags"] = self.allowed_tags
+        if self.allowed_attrs != MarkdownField.allowed_attrs:
+            kwargs["allowed_attrs"] = self.allowed_attrs
```

Comment:
I have a small concern about these three conditionals: they have a long-distance interaction with the defaults defined above on lines 83-85, such that if either change, the other should as well. Can we, should we, do we want to express that in code? Like `if not self.is_safe == IS_SAFE_DEFAULT` and use the same `IS_SAFE_DEFAULT` on line 83?

Reply:
I see your point, but I think these are unlikely to change and we'll scrutinize them if they do.

Reply:
if you have the custom Fields manual up while editing this code, you'll have the understanding that if you touch a constructor for a Field, you need to touch its deconstructor as well. 

But you're totally right, I do wish there was a better way to express the defaults for a django field when also using mutable collections. the default should probably be a frozenset too... i'll thnk about it...